src/EiffelEvents.Net/Schemas/EiffelActivityCanceledEvent/1.0.0.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
I see we added all schemas for all events, however the SDK does not support all of them, do we need these schemas, as they not used till now?